### PR TITLE
2027 - guard setters against negative values

### LINKED ISSUE #2027: Guard setters against negative values
Those public setters aren't supposed to be negative values and need to be guarded to avoid un-expected results.

1) [`BlendableAction#setTransitionLength(double)`](https://github.com/JMonkey-Archive/jmonkeyengine/blob/bbd041c1020b1e80154a5c78f9ae45513bcc977e/jme3-core/src/main/java/com/jme3/anim/tween/action/BlendableAction.java#L107-L110)
2) [`AbstractTween#setLength(double)`](https://github.com/JMonkey-Archive/jmonkeyengine/blob/bbd041c1020b1e80154a5c78f9ae45513bcc977e/jme3-core/src/main/java/com/jme3/anim/tween/AbstractTween.java#L59-L61)

I don't think duration factors can be negative.

### PR DESCRIPTION
**Pull request by SaichandChowdary**
Resolves #2027

### CODE REVIEW

**Comment by Scrappers-glitch** [comment]
Thanks for the PR, I will look into it soon.

File: jme3-core/src/test/java/com/jme3/anim/tween/action/ClipActionTest.java
Comment (Scrappers-glitch) [inline jme3-core/src/test/java/com/jme3/anim/tween/action/ClipActionTest.java:37]: Could you remove this white space, please?

**Review by Scrappers-glitch — CHANGES_REQUESTED** [review]

File: jme3-core/src/test/java/com/jme3/anim/tween/action/ClipActionTest.java
Comment (SaichandChowdary) [reply jme3-core/src/test/java/com/jme3/anim/tween/action/ClipActionTest.java:36]: done

**Review by Scrappers-glitch — APPROVED** [review]

File: jme3-core/src/test/java/com/jme3/anim/tween/action/ClipActionTest.java
Comment (Scrappers-glitch) [reply jme3-core/src/test/java/com/jme3/anim/tween/action/ClipActionTest.java:36]: Thanks, you can resolve this conversation.

**Comment by Scrappers-glitch** [comment]
I don't have anything to add to this PR, however, I am not sure if this change needs a unit test class, I will leave this to the maintainers to decide.

**Comment by stephengold** [comment]
Unless there's further substantive discussion, I plan to integrate this PR in about 24 hours.

**Pull request merged**
